skyrim-platform/src/platform_se/skyrim_platform/DevApi.cpp

```diff
@@ -26,8 +26,15 @@ JsValue DevApi::Require(const JsFunctionArguments& args,
     throw InvalidArgumentException("fileName", fileName);
   }
 
-  while (!fileName.empty() && (fileName[0] == '/' || fileName[0] == '\\')) {
-    fileName = { fileName.begin() + 1, fileName.end() };
+  while (!fileName.empty()) {
+    // Trying to get rid of absolute paths...
+    while (!fileName.empty() && (fileName[0] == '/' || fileName[0] == '\\')) {
+      fileName = { fileName.begin() + 1, fileName.end() };
+    }
+    // Trying to get rid of `DRIVELETTER:`
+    while (fileName.size() > 1 && fileName[1] == ':') {
+      fileName = { fileName.begin() + 2, fileName.end() };
+    }
   }
```

Comment:
BTW shouldn't it throw instead? If it finds paths to begin from `/` or `\`, or `C:` etc?